### PR TITLE
CCM-1062

### PR DESCRIPTION
**Pull request by TomHattersley**
## Summary
CCM-1062 Adds a link to the service page in the API documentation

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/communications-manager.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ info:
     * enrichment of recipient details
     * support for accessible formats
 
+    For more information about this services capabilities, see [Communications Manager Service](https://digital.nhs.uk/services/communications-manager).
```
Comment (PhilipSkinner) [inline specification/communications-manager.yaml:17]: Example review comment.

**Review by PhilipSkinner — CHANGES_REQUESTED** [review]
Changes requested

File: specification/communications-manager.yaml (hunk 1 above)
Comment (TomHattersley) [reply specification/communications-manager.yaml:17]: Resolved

**Review by PhilipSkinner — APPROVED** [review]
Approved

**Pull request merged**